src/content/architecture/high-availability/_index.en.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "High Availability"
+date = 2020-02-19T21:08:37+01:00
+weight = 6
++++
+## Introduction
+
+Submariner supports High Availability (Active/Passive) for the Gateway Engine component which is responsible
+for establishing secure tunnels to remote clusters. In order to use this feature, you will have to label more
+than a Node in your cluster with "submariner.io/gateway=true".
+On every node labeled with `submariner.io/gateway=true`, a Gateway Engine Pod is deployed and they perform
+a leader election process to determine the active instance for the cluster. The other Gateway Engine Pods
+await in standby mode ready to take over should the active instance fail.
+
+The Active Gateway Engine Pod creates an `Endpoint` CRD and updates it on the Broker cluster.
+The Route Agent Pods running on the local cluster learn about the `Endpoint`, create a VxLAN tunnel along with
+routing rules that point to the active Gateway Engine node for cross-cluster communication.
+
+![Figure 1 - High Availability in stable cluster](/images/high-availability/HA_Cluster1.png)
```

Comment:
Can we add in comments links to the source of the images (if available now), just in case we need to modify them later in time?

<!-- url.......  -->

Reply:
Ack. I created a new PPT that includes images used on website. Will update the URL as a comment.